### PR TITLE
[a11y][WPF] AutomationPeer customization support for TreeView and ListView

### PR DESCRIPTION
**Pull request by DmitriyKirakosyan**
@sevoku Please review.

### CODE REVIEW

**Comment by DmitriyKirakosyan** [comment]
This changes allow to setup AutomationPeer for TreeView and its children (TreeViewItem).
Client doesn't need to access to TreeViewItem in order to create automation peer for it. The idea here is that TreeView's automation peer has a specific method for its children peers creation:

https://github.com/mono/xwt/pull/901/files#diff-3d9a78c59e4d79eb6619e47a53e3b0a7R507

that can be implemented in client.

**Comment by DmitriyKirakosyan** [comment]
cc @BretJohnson ^

File: Xwt.XamMac/Xwt.Mac/AccessibleBackend.cs [hunk 1]
```diff
@@ -240,6 +241,16 @@ public void RemoveAllChildren ()
 			accessible.AccessibilityChildren = null;
 		}
 
+		public IEnumerable<object> GetChildren ()
+		{
+			var accessible = widget as INSAccessibility;
+			if (accessible == null) {
+				return new List<object> ();
```
Comment (sevoku) [inline Xwt.XamMac/Xwt.Mac/AccessibleBackend.cs]: `return Enumerable.Empty<object>();`

File: Xwt/Xwt.Accessibility/Accessible.cs [hunk 2]
```diff
@@ -289,6 +295,11 @@ public void RemoveAllChildren ()
 		{
 		}
 
+		public IEnumerable<object> GetChildren ()
+		{
+			return new List<object>();
```
Comment (sevoku) [inline Xwt/Xwt.Accessibility/Accessible.cs]: `return Enumerable.Empty<object>();`

File: Xwt.Gtk/Xwt.GtkBackend/AccessibleBackend.cs [hunk 3]
```diff
@@ -192,6 +193,12 @@ public virtual void RemoveAllChildren ()
 			// TODO
 		}
 
+		public virtual IEnumerable<object> GetChildren ()
+		{
+			// TODO
+			return new List<object> ();
```
Comment (sevoku) [inline Xwt.Gtk/Xwt.GtkBackend/AccessibleBackend.cs]: `return Enumerable.Empty<object>();`

**Comment by DmitriyKirakosyan** [comment]
@sevoku ^

File: Xwt/Xwt/TreeNavigator.cs [hunk 4]
```diff
@@ -115,6 +115,11 @@ public TreeNavigator AddChild ()
 			pos = backend.AddChild (pos);
 			return this;
 		}
+
+		public int GetChildrenCount ()
```
Comment (sevoku) [inline Xwt/Xwt/TreeNavigator.cs]: This doesn't seem to be related, can we move this into a separate commit pls?

Comment (DmitriyKirakosyan) [reply Xwt/Xwt/TreeNavigator.cs]: Removed. I think it is not needed anymore.
